orchagent/qosorch.cpp

```diff
@@ -148,6 +150,16 @@ task_process_status QosMapHandler::processWorkItem(Consumer& consumer, KeyOpFiel
                 freeAttribResources(attributes);
                 return task_process_status::task_failed;
             }
+            if ((qos_map_type_name == CFG_DSCP_TO_TC_MAP_TABLE_NAME) && (qos_object_name == DEFAULT_DSCP_TO_TC_MAP_NAME))
```

Comment:
Can you please remove it from the switch attribute when `DSCP_TO_TC_MAP|AZURE` is being removed?